js/constants.js

```diff
@@ -16,3 +16,7 @@ export const LYRICS_VIEW_ID = 'lyrics';
 export const TAB_CHANGED = 'TAB_CHANGED';
 
 export const VIDEO_ELEMENT_SELECTOR = 'video';
+
+export const MUTATION_OBSERVER_CONFIG = { childList: true };
+
+export const TITLE_SELECTOR = 'TITLE';
```

Comment:
If the title selector was lowercase it would be easier to parse it as a query selector.